DOCSP-33172 added meta description for top 250 pages in docs-node (#832)
mongodb/docs-node: source/usage-examples/findOne.txt

   :values: tutorial

.. meta:: 

>    :description: Learn how to find one document in your MongoDB collection by using the Node.js driver.

Reviewer: Suggestion:

Similar to the suggestions provided for source/usage-examples/find.txt, I think one of these would keep them similar to each other:

```suggestion
   :description: Learn how to retrieve one document from MongoDB by using the Node.js driver.
```

```suggestion
   :description: Follow a usage example of retrieving one document from MongoDB by using the Node.js driver.
```
